lib/Sema/TypeCheckProtocol.cpp

```diff
@@ -5069,7 +5069,8 @@ void TypeChecker::checkConformancesInContext(DeclContext *dc,
                         ? diag::redundant_conformance_conditional
                         : diag::redundant_conformance;
       diagnose(diag.Loc, diagID, dc->getDeclaredInterfaceType(),
-               diag.Protocol->getName());
+               diag.Protocol->getName())
+        .fixItRemove(diag.Loc);
```

Comment:
We don't usually attach fixits to errors directly.  Could you please emit a helpful note and attach it there?